File: src/main/java/it/contrader/dto/CocktailDTO.java
package it.contrader.dto;

import it.contrader.model.Cocktail.Cocktailtype;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor

public class CocktailDTO {
	private Long id;
	

	private String name;
	
	private String type;
	
	private boolean IsAlcohol;
	
	private int AVG;
	
	private String alcohol;

}

File: src/dto/cocktaildto.ts
import {Usertype} from './usertype';

/**
 * Classe DTO di User. DEVE essere uguale (stesso nome classe, stessi attributi e stessi nomi) a
 * quello nel backend. 
 * 
 * @see Usertype
 * 
 * @author Stefano Mazzeo
 */
export class CocktailDTO {

   id: number;

   name : string;

   type: string;

   isAlcohol : number;

   AVG : number;

   alcohol : string;

}

